fix-hosts-file-on-boot/templates/fix-hosts.service.j2

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Fix Hosts service
+
+[Service]
+Type=oneshot
+ExecStart={{ chia_root }}/fix-hosts.sh
+User=ubuntu
```

Comment:
Maybe just leave these out or specify root? No sense running as ubuntu to then just sudo to actually do anything